kernel/src/scheduler/mod.rs

```diff
@@ -67,7 +67,6 @@ pub(crate) fn schedule() -> ! {
                             // There is no process but we're the "main" thread,
                             // aggressively try and advance the replica
                             let start = rawtime::Instant::now();
-                            crate::nrproc::advance_all();
                             crate::arch::advance_fs_replica();
```

Comment:
Right now cnrfs isn't using dynamic replication, yes? Is that why advance_fs_replica() remains?

If so, I think that's just fine for now - no urgent need to port to cnrfs for the benchmarks we're looking at. 

I'm asking because while the advance_fs_replica() logic remains, the tlb work queues remain large/complicated (https://github.com/vmware-labs/node-replicated-kernel/issues/290). So it might be nice to document that implementing this for cnrfs would fix that issue :)